common/util.go

```diff
@@ -59,6 +59,9 @@ func IsDirectoryMounted(path string) bool {
 		return false
 	}
 
+	// removing trailing / from the path
+	path = strings.TrimRight(path, "/")
```

Comment:
Do we need a test for this?

Reply:
Creating one. Will raise a new PR for this.